.github/workflows/check-no-merge-label.yml

```diff
@@ -0,0 +1,25 @@
+name: check-no-merge-label
+
+permissions:
+  pull-requests: read
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, labeled, unlabeled, synchronize]
+    branches:
+      - 'main'
+      - 'release/**'
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check 'NO-MERGE' label
+      run: |
+        echo "Merging permission is disabled when the `NO-MERGE` label is applied."
```

Comment:
Ok this is backwards. When the label is applied, we need to go to the error. When it is not applied, we exit 0.

Will fix tomorrow.

Reply:
note that the backticks in the echo string mean that bash will try to run that as a command. this is a bug in the servicing-labels action too

Reply:
Oh wow, didn't know that. Thanks for noticing.

Reply:
That means we have a bug here! I'll include the fix in this PR:
https://github.com/dotnet/runtime/blob/36c79422b2300e15495bf1277d5e586998480f2d/.github/workflows/check-service-labels.yml#L18

cc @hoyosjs fyi